src/taipy/core/pipeline/pipeline.py

```diff
@@ -169,7 +169,7 @@ def _is_consistent(self) -> bool:
             is_data_node = not is_data_node
         return True
 
-    def _get_tasks(self) -> Dict[str, Task]:
+    def _get_tasks(self):
```

Comment:
There is no need to inherite this method.
```suggestion
    def _get_tasks(self)) -> Dict[str, Task]:
```